python/lsst/rapid/analysis/nightReport.py

```diff
@@ -98,11 +98,13 @@ class ColorAndMarker:
 
 class NightReporter():
 
-    def __init__(self, repoDir, dayObs, deferLoadingData=False):
+    def __init__(self, location, dayObs, deferLoadingData=False):
```

Comment:
Docstrings.

Reply:
This whole class is in desperate need of a rewrite I think. I'm going to defer writing docs for it till I do that, but will make a ticket number for the redo now, which will specifically mention the docs. Basically it's such an internal mess that I don't really want to document it!